src/ItemNote.ts

```diff
@@ -27,6 +27,9 @@ import { ensureFolderExists, getPocketItemPocketURL } from "./utils";
 
 const DEFAULT_ITEM_NOTES_FOLDER = "/";
 
+// Windows allows at-most 255 characters. We use a safely shorter length for additional compatibility.
+const MAXIMUM_TITLE_LENGTH = 255 - 15;
```

Comment:
Please change to "255 - 5", based on the reasoning in the review (max dup index is 1000, so 4 chars, and then there's a space separator).

Reply:
Accoridng to your calculation it should actually be 8 - because there are three more characters for the file extension `.md`.

I intentionally went a bit below, as I didn't want to be close to the threshold. In case something unexpected happens it would be good to have some extra space reserved. Because most people won't read more than a few words, I somewhat arbitrarily used 15 instead of 3.

I can add the your explanation and my comment and either set it to 8, 10, 12 or 15. What do you think?

Reply:
Oh you're right, good catch. I think you should go forward with the 8 chars option.